hvplot/converter.py

```diff
@@ -2173,11 +2173,14 @@ def _process_tiles_without_geo(self, data, x, y):
             if data.crs is not None:
                 data = data.to_crs(epsg=3857)
             return data, x, y
-        else:
+        elif not (is_dask(data) or is_ibis(data)):
```

Comment:
```suggestion
        elif not is_lazy_data(data):
```

So we can easy expand it in the future